guppylang/std/builtins.py

```diff
@@ -83,11 +83,11 @@ class nat:
 _n = TypeVar("_n")
 
 
-class array(Generic[_T, _n]):
+class array(list[_T], Generic[_T, _n]):
     """Class to import in order to use arrays."""
 
     def __init__(self, *args: Any):
-        pass
+        list.__init__(self, args)
```

Comment:
This is needed so that users can use the `array` constructor inside comptime functions

Reply:
include this as a comment in the code